### PR TITLE
Add British localization

### PR DESCRIPTION
**Pull request by Jerry23011**
As discussed in #498 
@Bexxia could you take a look at the spellings in [here](https://github.com/Dimillian/IceCubesApp/pull/577/files#diff-bc72a159ff337a63b3bc4799f2abd12f82e58e40ab87064025e20bd63c1c3b56)? It would be great if you could comment under the strings that need to be changed.

### CODE REVIEW

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+"settings.display.font" = "Timeline Font";
+"settings.display.font.system" = "System";
+"settings.display.font.custom" = "Custom";
+"settings.display.avatar.position" = "Avatar position";
+"settings.display.avatar.shape" = "Avatar shape";
+"settings.display.navigation-title" = "Display Settings";
+"settings.display.restore" = "Restore defaults";
+"settings.display.section.display" = "Display";
+"settings.display.section.theme" = "Theme";
+"settings.display.section.theme.footer" = "Custom colors can only be set if match system color scheme is disabled";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:69]: Custom colours can only be set if match system colour scheme is disabled.

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+"settings.display.status.action-buttons" = "Status action buttons";
+"settings.display.status.media-style" = "Status media style";
+"settings.display.translate-button" = "Show translate button";
+"settings.display.theme.background" = "Background color";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:73]: Background colour

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+"settings.display.theme.secondary-background" = "Secondary Background color";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:74]: Secondary Background colour

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+"settings.display.theme.systemColor" = "Match System";
+"settings.general.browser" = "Browser";
+"settings.general.browser.in-app" = "In-App Browser";
+"settings.general.browser.system" = "System Browser";
+"settings.general.display" = "Display Settings";
+"settings.general.instance" = "Instance Information";
+"settings.general.push-notifications" = "Push Notifications";
+"settings.general.remote-timelines" = "Remote Local Timelines";
+"settings.push.boosts" = "Boosts";
+"settings.push.favorites" = "Favorites";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:85]: Favourites

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+"account.edit.account-settings.private" = "Private";
+"account.edit.account-settings.section-title" = "Account settings";
+"account.edit.display-name" = "Display Name";
+"account.edit.error.save.message" = "Error while saving your profile, please try again.";
+"account.edit.error.save.title" = "Error while saving your profile";
+"account.edit.navigation-title" = "Edit Profile";
+"account.edit.post-settings.privacy" = "Default privacy";
+"account.edit.post-settings.section-title" = "Post settings";
+"account.edit.post-settings.sensitive" = "Sensitive content";
+"account.favorited-by" = "Favorited by";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:189]: Favourited by

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+"notifications.error.title" = "An error occurred";
+"notifications.label.favorite %lld" = "starred";
+"notifications.label.follow %lld" = "followed you";
+"notifications.label.mention %lld" = "mentioned you";
+"notifications.label.reblog %lld" = "boosted";
+"notifications.label.poll" = "poll ended";
+"notifications.label.follow-request" = "requested to follow you";
+"notifications.label.status" = "posted a status";
+"notifications.label.update" = "edited a post";
+"notifications.menu-title.favorite" = "Favorite";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:273]: Favourite

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,384 @@
...
+// MARK: Package: Status
+"status.action.translate" = "Translate";
+"status.action.translate-from-%@" = "Translate from %@";
+"status.action.translated-label" = "Translated using DeepL.com";
+"status.action.bookmark" = "Bookmark";
+"status.action.boost" = "Boost";
+"status.action.copy-text" = "Copy Text";
+"status.action.delete" = "Delete";
+"status.action.edit" = "Edit";
+"status.action.favorite" = "Favorite";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:301]: Favourite

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 8]
```diff
@@ -0,0 +1,384 @@
...
+"status.action.message" = "Message";
+"status.action.pin" = "Pin";
+"status.action.post" = "Post";
+"status.action.quote" = "Quote this post";
+"status.action.reply" = "Reply";
+"status.action.section.your-post" = "Your post";
+"status.action.share" = "Share this post";
+"status.action.unbookmark" = "Unbookmark";
+"status.action.unboost" = "Unboost";
+"status.action.unfavorite" = "Unfavorite";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:312]: Unfavourite

File: IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings [hunk 9]
```diff
@@ -0,0 +1,384 @@
...
+"status.action.unpin" = "Unpin";
+"status.action.view-in-browser" = "View in Browser";
+"status.draft.delete" = "Delete Draft";
+"status.draft.save" = "Save Draft";
+"status.editor.ai-prompt.correct" = "Correct text";
+"status.editor.ai-prompt.emphasize" = "Emphasize text";
```
Comment (Bexxia) [inline IceCubesApp/Resources/Localization/en-GB.lproj/Localizable.strings:318]: Emphasise text

**Comment by Jerry23011** [comment]
think we are ready 🚀

**Comment by Dimillian** [comment]
🇬🇧 let's go

**Comment by sothawo** [comment]
If your text for the PR contains 
> Closes #498 
the issue will be closed automatically when the PR is merged.

**Pull request merged**